### DISCUSSION TITLE
Is there any way to export to wordpress?

### DISCUSSION THREAD

**Discussion by lingywingy** [xenocrat/chyrp-lite#140] [Q&A] [answered]
I wanted to enable google adsense on my chyrp blog snd couldn’t find a way to do so, so now I’m planning on moving to wordpress. Is there any way for me to export to wordpress?

**Comment by xenocrat** [comment, marked as answer]
Hello there.

There's no specific Wordpress export option, but the exporter produces an XML text document that could be opened in a text editor and wrangled into a format Wordpress will accept.

Adsense is basically just a JavaScript injection, so I don't see any reason it can't be enabled on your Chyrp Lite blog. The most portable way of doing this would be to [create a simple module](https://github.com/xenocrat/chyrp-lite/wiki/Making-Your-First-Module) to respond to the `javascript` or `scripts` [triggers](https://github.com/xenocrat/chyrp-lite/wiki/Introduction-to-Triggers), but if you don't want to bother with that you can just paste the adsense JavaScript into your theme's template.

For example, if you are using Blossom theme you would simply add the code provided by Google [before this line](https://github.com/xenocrat/chyrp-lite/blob/master/themes/blossom/layouts/default.twig#L11) in _default.twig_.

**Reply by lingywingy** [reply]
I have tried inserting the code into the theme, but adsense doesn’t seem to be able to find it

**Comment by xenocrat** [comment]
After you make changes to a theme template, be sure to delete the contents of _includes/caches/twig_. This forces Twig to rebuild the page from the templates.
